src/patternfly/components/NotificationDrawer/examples/NotificationDrawer.md

```diff
@@ -39,7 +38,7 @@ cssPrefix: pf-c-notification-drawer
       2 unread
     {{/notification-drawer-header-status}}
     {{#> notification-drawer-header-action}}
-      {{#> dropdown id=(concat notification-drawer--id "-header-action") dropdown-menu--modifier="pf-m-align-right" dropdown--IsActionMenu="true" dropdown-toggle--modifier="pf-m-plain" dropdown--HasKebabIcon="true" aria-label="Actions"}}{{/dropdown}}
+      {{> dropdown dropdown--id=(concat notification-drawer--id "-header-action") dropdown-toggle--IsPlain="true"}}
```

Comment:
This still needs `dropdown-menu--modifier="pf-m-align-right"` - my comment before was to remove a duplicate because there were two.